src/pymor/analyticalproblems/functions.py

```diff
@@ -372,7 +373,67 @@ def evaluate(self, x, mu=None):
         return np.prod([f(x, mu) for f in self.functions], axis=0)
 
 
-class BitmapFunction(Function):
+class DataFieldFunction(Function):
+    """Define a 2D |Function| via a matrix that represents a data field.
```

Comment:
Is there such a thing as a 'data field'? I don't know a better name at the moment, but 'data field' sounds strange in my ears. @pymor/all, any other ideas?

Reply:
I also thought about this quite a lot, and also did not come up with something better... So I'd be happy about better suggestions.

Reply:
Maybe `BitmapFunction` is actually the best name. The data really is a bitmap or rather a [pix-map](https://en.wikipedia.org/wiki/Bitmap).
Ideally, a new `BitmapFunction` would get a `from_file` classmethod to construct the function from an image file. For backwards compatibility, one might consider allowing a filename instead of a data array in `__init__`. However, this usage should be deprecated.